drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -39,7 +38,6 @@ local SWITCH_INITIALIZED = "__switch_intialized"
 -- in the device table for devices that joined prior to this transition, and it
 -- will not be set for new devices.
 local COMPONENT_TO_ENDPOINT_MAP = "__component_to_endpoint_map"
-local BOUNDS_CHECKED = "__bounds_checked"
```

Comment:
It might be a good idea to include a change that deletes the field from persistent storage. That way if we later add some new code that happens that same field name we won't run into issues with stale data. It wouldn't need to live in the code forever, maybe just for a few weeks to give most hubs a chance to pick it up.

Reply:
Gotcha, thanks!